package.json

```diff
@@ -27,6 +27,7 @@
         "lodash": "^4.17.19",
         "postcss": "^8.4.14",
         "pusher-js": "^8.0.0",
+        "tailwind-scrollbar": "^2.1.0",
```

Comment:
This package is not mandatory, but it does feel well-supported. The alternative is doing this with vanilla CSS, which seems like a reimplementation of what this package does anyway.

https://openbase.com/js/tailwind-scrollbar